app/lib/actions/pulp3/repository/sync.rb

```diff
@@ -0,0 +1,60 @@
+module Actions
+  module Pulp3
+    module Repository
+      class Sync < Pulp3::AbstractAsyncTask
+        include Helpers::Presenter
+
+        input_format do
+          param :smart_proxy_id
+          param :pulp_id
+          param :task_id # In case we need just pair this action with existing sync task
+          param :source_url # allow overriding the feed URL
+          param :options # Pulp sync options
+        end
```

Comment:
I would rather define a plan method for this action, that takes arguments like `def plan(repository, smart_proxy, options)`, and plans those explicitly.    Also, Pulp3 repositories don't have a pulp_id set, so i'm wondering how this is working exactly :)  Unless its just relying on one with 'nil'

Reply:
So the pulp_id is still being set for repositories from here I believe: 
https://github.com/Katello/katello/blob/c317cce980982603f8c2ac78a955e329e87de497/app/models/katello/repository.rb#L182

I will change it to use repository id though..